src/component/material-ui/nav-wrapper/index.js

```diff
@@ -12,12 +14,17 @@ class NavWrapper extends Component {
       anchorEl: null,
       dialogOpen: true,
       popoverOpen: false,
+      viewingSaved: false,
     }
 
   handleToggle = () => {
     this.setState({ open: !this.state.open });
   }
 
+  toggleViewingSaved = () => {
+    this.setState({ viewingSaved: !this.state.viewingSaved });
```

Comment:
Same with this one:

```
this.setState(prevState => ({
  viewingSaved: !prevState.viewingSaved,
});
```

Reply:
This viewingSaved state is not longer needed, it was what i forgot to remove. I am using the location.pathname as the check for this logic now.

so the  
```
toggleViewingSaved = () => {
    this.setState({ viewingSaved: !this.state.viewingSaved });
  }
```

can be removed as well